src/org/apache/lucene/analysis/hunspell/Dictionary.m

```diff
@@ -1132,9 +1132,9 @@ void OrgApacheLuceneAnalysisHunspellDictionary_parseAffixWithJavaUtilTreeMap_wit
 }
 
 JavaNioCharsetCharsetDecoder *OrgApacheLuceneAnalysisHunspellDictionary_getJavaEncodingWithNSString_(OrgApacheLuceneAnalysisHunspellDictionary *self, NSString *encoding) {
-  if ([@"ISO8859-14" isEqual:encoding]) {
-    return create_OrgApacheLuceneAnalysisHunspellISO8859_14Decoder_init();
-  }
+//  if ([@"ISO8859-14" isEqual:encoding]) {
```

Comment:
This change makes sense, and we might be able to put that logic in `translate.py` – the script hides the `#inculde` of the decoder since it depends on some JDK 7 classes not available in J2ObjC's JRE emulation library, causing this init call to be a nuisance to the C compiler.